src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -1427,7 +1428,17 @@ public <V extends View> V configRoundtrip(V view) throws Exception {
         return view;
     }
 
-    public <C extends Cloud> C configRoundtrip (C cloud) throws Exception {
+    /**
+     * Performs a configuration round-trip testing for a cloud.
+     * The given cloud is added to the cloud list of Jenkins.
+     *
```

Comment:
```suggestion
     * <p>
```
(When you are used to Markdown, it is hard to remember that Javadoc is just HTML!)

Reply:
Sorry :) and thank you